eo-maven-plugin/src/test/java/org/eolang/maven/VerifyMojoTest.java

```diff
@@ -203,4 +240,30 @@ private static void applyXsl(final String xsl, final Path xml) throws Exception
         ).pass(new XMLDocument(xml));
         new HmBase(xml.getParent()).save(output.toString(), xml.getParent().relativize(xml));
     }
+
+    /**
+     * Parse the error message to program name and error line number for checking.
+     * @param logs Logs logs
+     * @param error String needed error message
+     */
+    private String parserMessage(final Logs logs, final String error) {
```

Comment:
@Yanich96 I strongly believe that it's not right to modify the log result to match the regular expression. 
You allow an ability to make a mistake in two places instead of one (regular expression). Also you allow user a one more spot to make changes in order the tests don't work.
I believe it would be much better not to touch the log and try to search matches in the unmodified message you get from exception